client/overview/style.scss

```diff
@@ -33,9 +33,24 @@
 	}
 
 	.wcpay-connection-success {
+		position: relative;
 		text-align: center;
 		padding: $gap-large;
-		padding-bottom: $gap;
+
+		&__dropdown {
+			position: absolute;
+			top: 0 + $gap-large;
+			right: 0 + $gap-large;
```

Comment:
Any reason to sum `0` here? 🤔

Reply:
Oh sorry, that was part of the debugging :D Removed https://github.com/Automattic/woocommerce-payments/pull/5992/commits/870eae5ea70dba84209676bc380cb11328197ebe